ui/src/templates/api/index.ts

```diff
@@ -440,3 +440,15 @@ export const updateStackName = async (stackID, name) => {
 
   return resp
 }
+
+export const fetchReadMe = async (directory: string) => {
+  const resp = await fetch(
+    `https://raw.githubusercontent.com/influxdata/community-templates/master/${directory}/README.md`
+  )
+
+  if (!resp.ok) {
+    throw new Error('Network response was not ok')
```

Comment:
do you know what `resp` looks like when this condition gets hit? is there any kind of error message we could throw here?

Reply:
ive got resp.statusText now added, it says it will be "A string (default value "OK"), which corresponds to the HTTP status code message."